packages/build/src/steps/run_steps.js

```diff
@@ -173,7 +167,6 @@ export const runSteps = async function ({
   }
 
   return {
-    stepsCount,
```

Comment:
I am not entirely sure if we can remove this. The `stepsCount` is used in some telemetry reports here: https://github.com/netlify/build/blob/main/packages/build/src/telemetry/main.js#L89

So either we are fine with removing the steps from telemetry or we keep counting and just do not display the index.

Reply:
Oh I missed that! I will add it back in 👍